.version

```diff
@@ -1 +1 @@
-1.5.0
+1.5.1
```

Comment:
I just merged my changes, this will need to go to 1.5.3

Reply:
Fixed